packages/sdk/src/utils/errors/errorHandling.ts

```diff
@@ -0,0 +1,166 @@
+// import { BigNumber, ethers, utils } from 'ethers';
```

Comment:
why? remove this empty file from project.

Reply:
+1

Reply:
the point was to use this when we integrate error handling, but it's not used yet so I removed it  [c36bf33](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/10/commits/c36bf330f7daf2dab92b10c2d0c1ef4b5e4e1cc0)